src/lib/helpers/date.ts

```diff
@@ -58,3 +58,30 @@ export const LocalizedDateTime: typeof DateTime = (() => {
     Settings.defaultZone = "Europe/Oslo";
     return DateTime;
 })();
+
+export const isClassExcludedByTimeFilter = (
+    _class: RezervoClass,
+    excludeClassTimeFilter: ExcludeClassTimeFilter,
+): boolean => {
+    if (
+        _class.startTime.weekday !== excludeClassTimeFilter.weekday &&
+        _class.endTime.weekday !== excludeClassTimeFilter.weekday
+    ) {
+        return false;
+    }
+
+    // Include a check for the middle value, in case of entirely overlapping exclude-filter
+    const middle = _class.startTime.plus(_class.endTime.minus(_class.startTime.toMillis()).toMillis() / 2);
```

Comment:
Why is it not sufficient to include the bounds (`>=`,`<=`)?

Reply:
Consider wanting to excluded classes 9-10, but want to include classes ending at 9 or starting at 10.
In order to included those, but not show classes starting exactly 9 and ending exactly 10, it is necessary to either check starting and ending separately, or use the same checking logic but with an additional middle value.
Either is possible, this just felt more readable (but that might be wrong) and shorter.

Reply:
Did not think of that, good catch 👍 Let's just roll with this 😎